README.md

```diff
@@ -118,7 +118,8 @@ Add or update the [app translations](https://github.com/FPGAwars/icestudio/tree/
   * use `scripts/update_translations.sh` to update current existing `*.po` language files based on the template (before editing them)
   * use `grunt compiletext` to regenerate `*.json` language files (these are what the IDE really need) to test them
 
-*Developer note*: use `export NWJS_BUILD_TYPE=sdk` after npm install to enable the chrome web console.
```

Comment:
There is no such thing as NWJS_BUILD_TYPE any more? It's not found anywhere including in these notes: https://nwjs.io/blog/v0.83.0